keyboards/jj_ergo/matrix.c

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2013 Oleg Kostyuk <cub.uanic@gmail.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "matrix.h"
```

Comment:
This should be converted to matrix lite, rather than the full custom matrix.

See the ergodox EZ's matrix.s file for details, as this board is based on that one.